lib/matplotlib/texmanager.py

```diff
@@ -196,6 +201,15 @@ def _get_preamble(self):
             # Custom packages (e.g. newtxtext) may already have loaded textcomp
             # with different options.
             _usepackage_if_not_loaded("textcomp"),
+            r"\pagestyle{empty}",
+            r"\begin{document}",
+            r"% The empty hbox ensures that a page is printed even for empty ",
+            r"% inputs, except when using psfrag which gets confused by it.",
+            r"\fontsize{%f}{%f}%%" % (fontsize, baselineskip),
+            r"\ifdefined\psfrag\else\hbox{}\fi%",
```

Comment:
Not sure if needed, but dropped a `%`:
```suggestion
            r"\ifdefined\psfrag\else\hbox{}\fi%%",
```

Reply:
Yes.

Reply:
Actually there wasn't because this is not %-formatted, but I agree the double %s are a bit confusing.  Repetition of braces (for f-strings) is a bit annoying too, but I went this route.

Reply:
Ah right, seems troublesome either way, but this is okay.